pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/realtime/segment/SegmentFlushThresholdComputer.java

```diff
@@ -65,6 +70,26 @@ public int computeThreshold(StreamConfig streamConfig, CommittingSegmentDescript
     long optimalSegmentSizeBytesMin = desiredSegmentSizeBytes / 2;
     double optimalSegmentSizeBytesMax = desiredSegmentSizeBytes * 1.5;
 
+    double segmentSizeVariation = DEFAULT_FLUSH_THRESHOLD_VARIANCE_PERCENTAGE;
```

Comment:
Compute this value within `StreamConfig` similar to other variables, e.g. `_flushThresholdSegmentSizeBytes`. The constants should sit in `StreamConfigProperties`

Reply:
done